tests/integration/suite/scheduler/metrics/daprconnections.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2024 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	"github.com/dapr/dapr/tests/integration/framework/process/grpc/app"
+	"github.com/dapr/dapr/tests/integration/framework/process/scheduler"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(daprconnections))
+}
+
+type daprconnections struct {
+	scheduler *scheduler.Scheduler
+	daprdA    *daprd.Daprd
+	daprdB    *daprd.Daprd
+	daprdC    *daprd.Daprd
+}
+
+func (c *daprconnections) Setup(t *testing.T) []framework.Option {
+	c.scheduler = scheduler.New(t)
+	srv := app.New(t)
+
+	c.daprdA = daprd.New(t,
+		daprd.WithSchedulerAddresses(c.scheduler.Address()),
+		daprd.WithAppProtocol("grpc"),
+		daprd.WithAppPort(srv.Port(t)),
+		daprd.WithAppID("A"),
```

Comment:
Please try not to use constant app IDs because it causes issues with cache mDNS entries. Use `daprd.AppID()` to get the UUID which is generated in the framework.

Reply:
since Im not using the logline htis can be removed, I just forgot to rm it earlier. Its removed now